### PR TITLE
Add command line flag for --exclude-file.

### PR DESCRIPTION
**Pull request by jalawrence**
Works like a series of `--exclude`s; sort of a counterpart to --include.

The rationale is to allow a file under developer control to specify exclusion patterns separate from fpm execution, which is under CI control (in our world).

### CODE REVIEW

**Comment by ahammond** [comment]
+1 this would be a nice feature

File: lib/fpm/command.rb [hunk 1]
```diff
@@ -346,6 +351,22 @@ def execute
       end
     end
 
+    # If --exclude-file was specified, read it as a file and append to
+    # the exclude pattern list.
+    if !exclude-file.nil?
```
Comment (jordansissel) [inline lib/fpm/command.rb]: You probably meant `exclude_file` not `exclude-file` (probably confusing if you're looking at the flag name, but `exclude-file` in ruby means 'subtract `file` from `exclude` variable')

File: lib/fpm/command.rb [hunk 2]
```diff
@@ -346,6 +351,22 @@ def execute
...
+      if !File.exists?(exclude-file)
```
Comment (jordansissel) [inline lib/fpm/command.rb]: same here, `exclude_file`

File: lib/fpm/command.rb [hunk 3]
```diff
@@ -139,6 +139,11 @@ def help(*args)
     excludes << val
     next excludes
   end # -x / --exclude
+
+  option "--exclude-file", "EXLCUDE_PATH",
```
Comment (jordansissel) [inline lib/fpm/command.rb]: typo, fix: `EXCLUDE_PATH`

**Comment by jordansissel** [comment]
Small typos in the code. Otherwise this looks OK to me. Let me know when you've fixed the typos and I'll take another look!

Thank you for helping make fpm more awesome :)

**Comment by jordansissel** [comment]
Merged with #906

**Pull request merged**